client/apps/Admin/src/pages/AssignmentsPage.tsx

```
@@ -0,0 +1,453 @@
+                        {isEditing && (
+                            <button
+                                className="assignment-badge assignment-badge--add"
+                                onClick={(e) => {
+                                    e.stopPropagation();
+                                    setAddingEmployeeToTaskId(info.row.original.taskId);
+                                }}
+                                title="הוסף עובד"
+                            >
+                                +
+                            </button>
+                        )}
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)